spec/features/proposal/proposal_progress_spec.rb

```diff
@@ -1,131 +1,245 @@
 describe 'Viewing Progress Page for Collection Metadata Proposals', js: true do
-  before do
-    login
-    set_as_proposal_mode_mmt(with_required_acl: true)
-    @collection_draft_proposal = create(:empty_collection_draft_proposal)
-  end
-
-  context 'when viewing an incomplete proposal in work' do
+  context 'when viewing the progress page as a user' do
```

Comment:
Again, diffing poorly.  I basically doubled the tests here since the approver sees different text on the progress page.